cli/src/plz/cli/composition_operation.py

```diff
@@ -5,21 +5,26 @@
 
 
 class CompositionOperation(Operation):
-    def _run_composition(self,
-                         composition: dict,
-                         composition_path: [(str, Any)]):
+    def _run_composition(
+            self,
+            composition: dict,
+            composition_path: [(str,
+                                Any)]):
         if 'indices_to_compositions' in composition:
             for index, c in composition['indices_to_compositions'].items():
                 self._run_composition(
                     composition=c,
-                    composition_path=composition_path + [
-                        ('parallel', index)])
+                    composition_path=composition_path + [('parallel',
+                                                          index)])
```

Comment:
@SamirTalwar  it seems that `split_all_comma_separated_values` triggers things like this (`index)])` fits in the previous line). Do you still want it?